start.py

```diff
@@ -345,10 +346,21 @@ def main(args):
 
                     # If using secret key (Note: key files must be DER-encoded)
                     'tokenSecretKey': f'file://{secret_key_file_path}',
+                    'proxyRoles': 'proxyadmin',
+                    'superUserRoles': 'proxyadmin,admin'
                 })
                 node.put_file(BROKER_CONF, PropertiesFile.dumps(broker_properties))
                 node.put_file(secret_key_file_path, secret_key_file)
 
+            for node in nodes:
```

Comment:
Is it possible to combine some of these loops over the nodes so that we aren't getting out and writing to the same files repeatedly? It seems super inefficient.